src/components/organization/PartnerForm.js

```diff
@@ -0,0 +1,108 @@
+import {Button, Grid, makeStyles, TextField} from "@material-ui/core";
+import {Controller, useForm} from "react-hook-form";
+import {validateEmail} from "../../utils/functions";
+import React, {useState} from "react";
+import {withSwal} from "../formsCommons/WithSwal";
+import {organizationsAPI} from "../../utils/apis/OrganizationApi";
+
+const useStyles = makeStyles((theme) => ({
+    button: {
+        height: '97%'
```

Comment:
Whaaaaat? XDDDDD

Reply:
w formie do skilli jest identycznie xd

Reply:
To ja tak chyba doszedłem do takiej heurystyki xDDD